packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.stories.ts

```diff
@@ -158,13 +162,14 @@ export const WithMessage = {
 } satisfies Story;
 
 /**
- * Multiselect listbox
+ * Multiselect listbox. You can disable the `Select all` option in the settings.
```

Comment:
```suggestion
 * Multiselect listbox. You can disable the `Select all` option by removing the `withCheckAll` property.
```

"Settings" might be a confusing term here